src/sentry/middleware/integrations/parsers/gitlab.py

```diff
@@ -45,11 +45,17 @@ def get_integration_from_request(self) -> Integration | None:
         try:
             _view, _args, kwargs = resolve(self.request.path)
             # Non-webhook endpoints
-            if "integration_id" in kwargs and "organization_slug" in kwargs:
-                self._integration = Integration.objects.filter(
-                    id=kwargs["integration_id"],
-                    organization_slug=kwargs["organization_slug"],
-                ).first()
+            if "integration_id" in kwargs and "organization_id_or_slug" in kwargs:
+                if str(kwargs["organization_id_or_slug"]).isdecimal():
+                    self._integration = Integration.objects.filter(
+                        id=kwargs["integration_id"],
+                        organization_id=kwargs["organization_slug"],
```

Comment:
I was checking this out but couldn't find where this `organization` is coming from for the `Integration` model

Reply:
https://github.com/getsentry/sentry/pull/70402